yarn-project/noir-contracts/src/contracts/inclusion_proofs_contract/src/main.nr

```diff
@@ -77,22 +86,18 @@ contract InclusionProofs {
 
     // Creates a value note owned by `owner`.
     #[aztec(private)]
-    fn create_note(
-        owner: AztecAddress,
-        value: Field,
-    ) {
+    fn create_note(owner: AztecAddress, value: Field) {
         let owner_private_values = storage.private_values.at(owner.to_field());
         let mut note = ValueNote::new(value, owner);
         owner_private_values.insert(&mut note, true);
     }
 
-
     // Proves that the owner owned a ValueNote at block `block_number`.
     #[aztec(private)]
     fn test_note_inclusion_proof(
         owner: AztecAddress,
         block_number: u32, // The block at which we'll prove that the note exists
-        spare_commitment: Field, // This is only used when the note is not found --> used to test the failure case
+        spare_commitment: Field // This is only used when the note is not found --> used to test the failure case
```

Comment:
It can be a little hard to follow exactly why it is there at first, if you extend the comment a bit that would be nice. Maybe just me that was slow.

Reply:
It was a bit out of context comment. Addressed in https://github.com/AztecProtocol/aztec-packages/pull/3680/commits/95634100dd8c2ce58611eeb92f480c53139c44dc